### DISCUSSION TITLE
Video.seekBy() is inconsistent, undocumented currentTime property works better

### DISCUSSION THREAD

**Discussion by dorukozerr** [expo/expo#33684] [Proposals and Ideas]
When using `useVideoPlayer` hook, `player.seekBy(targetSecond)` doesn't seek to the exact time specified - it consistently overshoots or undershoots the target position.

However, directly setting `player.currentTime = targetSecond` works perfectly and provides precise seeking.

Questions:
1. Why is seekBy() behavior inconsistent?
2. Why isn't the more reliable currentTime property documented?

Reproduction:
```js
const player = useVideoPlayer(videoUri, setupCallback)

// Inconsistent
player.seekBy(30) // Doesn't go to exactly 30s

// Works perfectly
player.currentTime = 30 // Goes to exactly 30s

**Comment by dorukozerr** [comment]
I missed that out in the docs. My bad it exists 🐱

**Comment by tsapeta** [comment]
I'd like to note that `seekBy(30)` is not the same as `currentTime = 30`. It's the same as `currentTime += 30` 😉 

If you are talking about iOS, then yeah, I think `seekBy` doesn't specify the tolerance, meaning that the platform can choose a bit different frame to seek to, depending on what is currently buffered. As a result, it may seek to another frame that is close to this time but is faster to seek to. I think using this function makes more sense when you want to have controls that let the user move backward/forward by let's say 10s. In this case, the user doesn't really care which frame he gets, it's close to 10s and that's fine.

**Reply by dorukozerr** [reply]
I totally missed that in the documentation, my bad 😸

**Reply by tsapeta** [reply]
Just wanted to note that I've clarified this a bit in the docs here: https://github.com/expo/expo/pull/33704

**Comment by maroparo** [comment]
I'd expect this to work with negative values as well, but doesn't seem to do so. It only seeks forward.
